x-pack/plugins/apm/public/components/app/mobile/transaction_overview/transaction_overview_tabs/stats_list/get_columns.tsx

```diff
@@ -0,0 +1,171 @@
+          detailedStatistics?.previousPeriod?.[name]?.latency;
+
+        const { currentPeriodColor, previousPeriodColor } = getTimeSeriesColor(
+          ChartType.LATENCY_AVG
+        );
+
+        return (
+          <SparkPlot
+            color={currentPeriodColor}
+            isLoading={detailedStatisticsLoading}
+            series={currentPeriodTimeseries}
+            valueLabel={asMillisecondDuration(latency)}
+            comparisonSeries={
+              comparisonEnabled && isTimeComparison(offset)
+                ? previousPeriodTimeseries
+                : undefined
+            }
+            comparisonSeriesColor={previousPeriodColor}
+          />
+        );
+      },
+    },
+    // throughput
+    {
+      field: 'throughput',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.throughputColumnAvgLabel',
+        { defaultMessage: 'Throughput' }
+      ),
+      align: RIGHT_ALIGNMENT,
+      render: (_, { throughput, name }) => {
+        const currentPeriodTimeseries =
+          detailedStatistics?.currentPeriod?.[name]?.throughput;
+        const previousPeriodTimeseries =
+          detailedStatistics?.previousPeriod?.[name]?.throughput;
+
+        const { currentPeriodColor, previousPeriodColor } = getTimeSeriesColor(
+          ChartType.THROUGHPUT
+        );
+
+        return (
+          <SparkPlot
+            color={currentPeriodColor}
+            isLoading={detailedStatisticsLoading}
+            series={currentPeriodTimeseries}
+            valueLabel={asTransactionRate(throughput)}
+            comparisonSeries={
+              comparisonEnabled && isTimeComparison(offset)
+                ? previousPeriodTimeseries
+                : undefined
+            }
+            comparisonSeriesColor={previousPeriodColor}
+          />
+        );
+      },
+    },
+    // launch time
+    ...(agentName && showAppLaunchTimeColumn
+      ? [
+          {
+            field: 'appLaunchTime',
+            name: i18n.translate(
+              'xpack.apm.mobile.transactions.overview.table.appLaunchTime',
+              {
+                defaultMessage: 'App launch time',
+              }
+            ),
+            align: RIGHT_ALIGNMENT,
+            render: (_, { appLaunchTime }) => {
+              return (
+                <EuiText size="s" textAlign="right">
+                  {asMillisecondDuration(appLaunchTime)}
+                </EuiText>
+              );
+            },
+          } as ITableColumn<MobileMainStatisticsByFieldItem>,
+        ]
+      : []),
+    // crash rate
+    {
+      field: 'crashRate',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.crashRate',
+        {
+          defaultMessage: 'Crash rate',
+        }
+      ),
+      align: RIGHT_ALIGNMENT,
+      render: (_, { crashRate }) => {
+        return (
+          <EuiText size="s" textAlign="right">
+            {crashRate.toFixed(2) + '%'}
```

Comment:
I got an error

Error: Cannot read properties of null (reading 'toFixed')

Reply:
Will it make sense to return a default value from the backend so we don't fall into that scenario?
From the typings I see `crashRate` is not nullable

Reply:
it seems asPercent func will handle it.   https://github.com/elastic/kibana/blob/32371a4a8c8d4d7f51098e74c59682ecea3f5bb9/x-pack/plugins/apm/common/utils/formatters/formatters.ts#L37-L56

Reply:
I expect the crash rate to behave similarly to the failed transaction rate

Reply:
Addressed in [65ea0d7](https://github.com/elastic/kibana/pull/153044/commits/65ea0d775ffb1416f27c4fa1d4f0119a45189467).